scripts/makeccTLDSet.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
This script will probably be broken by the time we need it again, however it will be useful to reference when we need it.